web-console/knikubevirt/wizard-storage/wizard-storage.md

```diff
@@ -0,0 +1,62 @@
+# Create Virtual Machine Wizard - Storage Step
```

Comment:
@yfrimanm - One more thing I noticed that's missing is a link from the README file to this new documentation.